.github/actions/pylint/index.js

```diff
@@ -0,0 +1,120 @@
+const core = require('@actions/core');
+const exec = require('@actions/exec');
+
+/**
+ * Execute pylint and report issues, if any found.
+ */
+async function run() {
+    let pylintOutput = "";  // Pylint process output.
+    const pylintDisable = core.getInput('pylint-disable') || "";
+    const pylintRCFile = core.getInput('pylint-rcfile') || "";
+    const pylintPaths = core.getInput('pylint-paths');
+
+    try {
```

Comment:
[fix] Be good to add a little note here on the expected exception/error handling from `exec.exec` as its not immediately obvious from the docs. As JS isn't our every day language, would help future maintainers.

Reply:
Added a comment in d3e8a3f